pkg/services/forwarder/ports.go

```diff
@@ -175,9 +169,12 @@ func (f *PortsForwarder) Expose(protocol types.TransportProtocol, local, remote
 				log.Error(err)
 			}
 		}()
-
+		proto := "unix"
+		if protocol == types.NPIPE {
+			proto = "npipe"
+		}
```

Comment:
Looks like you could use `protocol` directly instead of adding an intermediate `proto` variable?